articles/azure-monitor/insights/container-insights-live-logs.md

```diff
@@ -70,6 +72,9 @@ AKS can be configured to use Azure Active Directory (AD) for user authentication
 -  **Redirect URI**: Two **Web** application types need to be created. The first base URL value should be `https://afd.hosting.portal.azure.net/monitoring/Content/iframe/infrainsights.app/web/base-libs/auth/auth.html` and the second base URL value should be `https://monitoring.hosting.portal.azure.net/monitoring/Content/iframe/infrainsights.app/web/base-libs/auth/auth.html`.
 - After registering the application, from the **Overview** page select **Authentication** from the left-hand pane. On the **Authentication** page, under **Advanced settings** implicitly grant **Access tokens** and **ID tokens** and then save your changes.
 
+>[!NOTE]
+>If you're using this feature in Mooncake then the first base URL value should be `https://afd.hosting.azureportal.chinaloudapi.cn/monitoring/Content/iframe/infrainsights.app/web/base-libs/auth/auth.html` and the second base URL value should be `https://monitoring.hosting.azureportal.chinaloudapi.cn/monitoring/Content/iframe/infrainsights.app/web/base-libs/auth/auth.html`.
```

Comment:
Again, need to remove internal term "Mooncake" and replace with Azure China region.  Suggest rewording sentence to - If you are using this feature in Azure China region, the first base URL value should be `URL`, and the second base URL should be `URL`.